system/jlib/jtrace.cpp

```diff
@@ -220,8 +229,7 @@ class JLogSpanExporter final : public opentelemetry::sdk::trace::SpanExporter
 
                 if (hasMask(logFlags, SpanLogFlags::LogSpanDetails))
                 {
-                    out.appendf(", \"status\": \"%s\"", spanStatusToString(span->GetStatus()));
-                    out.appendf(", \"kind\": \"%s\"", spanKindToString(span->GetSpanKind()));
+
                     const char * description = span->GetDescription().data();
```

Comment:
Now duplicated above - should delete one of them

Reply:
this, along other obvious syntax issues were somehow lost, possibly after rebase.